grails-app/services/org/pih/warehouse/inventory/InventoryService.groovy

```diff
@@ -2625,10 +2625,12 @@ class InventoryService implements ApplicationContextAware {
     String exportBaselineQoH(products, quantityMapByDate) {
         def csvrows = []
         NumberFormat numberFormat = NumberFormat.getNumberInstance()
+        def session = RequestContextHolder.currentRequestAttributes().getSession()
```

Comment:
what is it added for? I don't see it being used anywhere in this file

Reply:
+1